sql/moz-fx-data-shared-prod/apple_ads_external/ios_app_campaign_stats_v1/metadata.yaml

```diff
@@ -4,14 +4,19 @@ description: |-
 owners:
 - kwindau@mozilla.com
 labels:
-  incremental: false
+  incremental: true
   owner1: kwindau@mozilla.com
 scheduling:
   dag_name: bqetl_ios_campaign_reporting
   depends_on_past: false
-  date_partition_parameter: null
 bigquery:
-  time_partitioning: null
+  time_partitioning:
+    type: day
+    field: date
```

Comment:
I'm not sure we can apply date partitioning to an existing table, we may need to recreate the table for this to work...

Reply:
Yeah I think that makes sense - it's not a long running backfill so I think it would be fine to delete, re-create, and backfill - just let Richard know ahead of time so he's aware.